.pre-commit-config.yaml

```
@@ -4,6 +4,11 @@ repos:
     hooks:
     - id: black
   - repo: https://github.com/pycqa/isort
-    rev: 5.8.0  # Also stored in dev-requirements.txt; update both together!
+    rev: 5.10.1  # Also stored in dev-requirements.txt; update both together!
     hooks:
     - id: isort
+  - repo: https://github.com/shellcheck-py/shellcheck-py
+    rev: v0.8.0.4
```

This needs a comment similar to the others in this file stating that the version needs to be kept in sync with `dev-requirements.txt`
```suggestion
    rev: v0.8.0.4  # Also stored in dev-requirements.txt; update both together!
```